packages/convenient_test_common_dart/lib/src/consts.dart

```diff
@@ -1,8 +1,9 @@
-const kConvenientTestManagerHost = '127.0.0.1';
-const kConvenientTestManagerPort = 3579;
+const String kConvenientTestManagerHost = String.fromEnvironment('MANAGER_HOST', defaultValue: '127.0.0.1');
```

Comment:
To be consistent w/ all other env variables, such as `CONVENIENT_TEST_APP_CODE_DIR`, maybe we should prefix all env var name with "CONVENIENT_TEST_"

Reply:
I did have it that way but it does make the launch command very long. Ex: `flutter run ./integration_test/main_test.dart --host-vmservice-port 9753 --disable-service-auth-codes --dart-define=CONVENIENT_TEST_APP_CODE_DIR="$($PWD | select -Expand Path)" --dart-define=CONVENIENT_TEST_MANAGER_HOST=10.0.2.2`

If you are good with that I will make the switch.